lerna.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     "packages/*"
   ],
-  "version": "15.3.0",
+  "version": "15.3.1",
```

Comment:
Don't change this manually. It is done by deploy scripts.

Reply:
Fixed